.setup/install_submitty/get_globals.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -ve
+
+# We assume a relative path from this repository to the installation
+# directory and configuration directory.
+SETUP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/.."
+SUBMITTY_CONFIGURATION_DIR="${SETUP_DIR}/../../../config"
+
+IS_VAGRANT=0
+if [ -d "${SETUP_DIR}/../.vagrant" ]; then
+    IS_VAGRANT=1
+fi
+
+IS_UTM=0
+if [ -d "${SETUP_DIR}/../.utm" ]; then
+    IS_UTM=1
+fi
+
+IS_CI=0
+if [ -f "${SETUP_DIR}/../.github_actions_ci_flag" ]; then
+    IS_CI=1
+fi
+
+SUBMITTY_REPOSITORY=$(jq -r '.submitty_repository' "${SUBMITTY_CONFIGURATION_DIR}/submitty.json")
```

Comment:
And then here, let's do an "assert" that the variable SUBMITTY_INSTALL_DIR we've defined above matches the value stored in this config file.  If somehow the value doesn't match, crash installation with an appropriate error.

Reply:
Sounds good